packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -1,12 +1,19 @@
-import React from 'react';
+import React, {
+  useCallback,
+  useReducer,
+  useState,
+  useEffect,
+  useRef,
+} from 'react';
 import PropTypes from 'prop-types';
 import Button from 'terra-button';
 import IconCalendar from 'terra-icon/lib/icon/IconCalendar';
 import Input from 'terra-form-input';
 import { injectIntl, intlShape } from 'react-intl';
 import classNames from 'classnames/bind';
 import ThemeContext from 'terra-theme-context';
-
+import VisuallyHiddenText from 'terra-visually-hidden-text';
+import * as KeyCode from 'keycode-js';
```

Comment:
Lets only import the keycodes that are used

Reply:
Changed here : https://github.com/cerner/terra-framework/pull/1208/commits/80162e0e8172a99f65f5de3b9813ac2f3b8bede5